tests/Console/Command/AddPrefixCommandTest.php

```diff
@@ -0,0 +1,254 @@
+<?php
+
+/*
+ * This file is part of the webmozart/php-scoper package.
+ *
+ * (c) Bernhard Schussek <bschussek@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Humbug\PhpScoper\Console\Command;
+
+use PHPUnit\Framework\Assert;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\ApplicationTester;
+use Symfony\Component\Filesystem\Filesystem;
+use function Humbug\PhpScoper\makeTempDir;
+
+/**
+ * @covers \Humbug\PhpScoper\Console\Command\AddPrefixCommand
+ */
+class AddPrefixCommandTest extends TestCase
+{
+    const FIXTURES_DIR = __DIR__.DIRECTORY_SEPARATOR.'..'.DIRECTORY_SEPARATOR.'..'.DIRECTORY_SEPARATOR.'../fixtures';
+    
+    /**
+     * @var ApplicationTester
+     */
+    private $appTester;
+
+    /**
+     * @var string
+     */
+    private $cwd;
+
+    /**
+     * @var string
+     */
+    private $tempDir;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUp()
+    {
+        if (file_exists($this->tempDir)) {
+            chdir($this->tempDir);
+        }
+
+        if (null !== $this->appTester) {
+            return;
+        }
+
+        $this->cwd = getcwd();
+
+        $application = new Application('php-scoper-test');
+        $application->addCommands([
+            new AddPrefixCommand(),
+        ]);
+        $application->setAutoExit(false);
```

Comment:
`$application->setCatchExceptions(false);` to not swallow exceptions where these would normally bubble up to output in use.